code/edge/src/main/java/com/adobe/marketing/mobile/EdgeHitProcessor.java

```diff
@@ -151,6 +152,28 @@ public void onComplete() {
 			edgeHit.getRequestId()
 		);
 
+		if (!UrlUtils.isValidUrl(url)) {
+			Log.warning(
+				LOG_TAG,
+				LOG_SOURCE,
+				"Unable to send network request with id (%s) as url is malformed, '%s'.",
+				edgeHit.getRequestId(),
```

Comment:
For these two logs, the requestId is a new uuid and will not make much sense for debugging, what do you think about logging the entityId/event id being processed instead?